src/core/app/init.ts

```diff
@@ -34,17 +39,32 @@ export async function appInit() {
     appConfigResponse.user = defaultUserProfileData;
   }
 
-  setAppLanguage(appConfigResponse.user.language);
+  const language = appConfigResponse.user.language;
+  setAppLanguage(language);
+
+  const defaultLayers = await getDefaultLayers(initialState.app, language);
+
+  const { basemapLayerId, basemapLayerUrl } = getBasemapFromDetails(defaultLayers);
 
   if (initialState.layers === undefined) {
-    initialState.layers = await getDefaultLayers(initialState.app);
+    initialState.layers = defaultLayers?.map((l) => l.id) ?? [];
   } else {
     // HACK: Remove KLA__ prefix from layers ids coming from url
     initialState.layers = initialState.layers.map((l) => l.replace(/^KLA__/, ''));
+
+    const basemapInUrl = findBasemapInLayersList(initialState.layers);
+    if (!basemapInUrl) {
+      initialState.layers.push(basemapLayerId);
+    }
```

Comment:
not clear what happens here:
Why we looking for all base map layers, but enable only one from them?